### PR TITLE
Configuring for buildout-recipe

### PR DESCRIPTION
**Pull request by icemac**
~~All tests are failing locally – maybe because I am running them on MacOS, let's see how GHA behaves.~~

Update tests to run locally, and on GHA.

### CODE REVIEW

**Comment by icemac** [comment]
Locally (MacOS) the tests fail with:

```
File ".../src/zdaemon/tests/tests.py", line 178, in zdaemon.tests.tests.test_kill
Failed example:
    system("./zdaemon -Cconf kill CONT")
Expected:
    kill(NNN, 18)
    signal SIGCONT sent to process NNN
Got:
    kill(NNN, 19)
    signal SIGCONT sent to process NNN



Failure in test test_logreopen (zdaemon.tests.tests)
Failed doctest test for zdaemon.tests.tests.test_logreopen
  File ".../src/zdaemon/tests/tests.py", line 192, in test_logreopen

----------------------------------------------------------------------
File ".../src/zdaemon/tests/tests.py", line 209, in zdaemon.tests.tests.test_logreopen
Failed example:
    system("./zdaemon -Cconf logreopen")
Expected:
    kill(NNN, 12)
    signal SIGUSR2 sent to process NNN
Got:
    kill(NNN, 31)
    signal SIGUSR2 sent to process NNN
```

Should I adapt the ReNormalizer? Is the second number actually important?

**Comment by dataflake** [comment]
@icemac The numeric representation of a signal depends on the platform you're on. So those numbers are not relevant for the test and should not be specified. There's more places where text output is expected that is tied to one specific platform.

File: .coveragerc [hunk 1]
```diff
@@ -1,11 +1,30 @@
+# Generated from:
```
Comment (dataflake) [inline .coveragerc:1]: I am confused. Weren't we killing off .coveragerc?

**Review by dataflake — APPROVED** [review]

File: .coveragerc (hunk 1 above)
Comment (icemac) [reply .coveragerc:1]: Generally, yes. But running coverage using multiple processes requires to have a `.coveragerc` it does not read its configuration from `tox.ini` in this case. So we have to keep `.coveragerc` for buildout recipes and packages which run tests in separate processes.

**Comment by icemac** [comment]
Thank you for reviewing this PR. 😃

**Pull request merged**